Monika After Story/game/0utils.rpy

```diff
@@ -703,25 +703,16 @@ python early in mas_utils:
         return False
 
 
-    def _get_version(ver_str, nums_only, split_nums=False):
+    def _get_version_nums(ver_str):
         """
-        Gets the version from a ver str
+        Gets version numbers from a version string
 
         IN:
             ver_str - version string to get version from
-            nums_only - True to only get ver numbers, False to get full ver
 
-        RETURNS: version str
+        RETURNS: version numbers as a list of ints
         """
-        if nums_only:
-            ver_nums = ver_str.partition("-")[0]
-        
-            if split_nums:
-                return ver_nums.split(".")
-
-            return ver_nums
-
-        return ver_str
+        return map(int, ver_str.partition("-")[0].split("."))
```

Comment:
I think we should explicitly convert into a `tuple` here, in py3 `map` returns a `Generator`.

Reply:
made it a list because compareVersionLists says lists explicitly.